metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -151,7 +151,7 @@ export function getNodeAutoInstrumentations(
 
   if (enabledInstrumentationsFromEnv && disabledInstrumentationsFromEnv) {
     diag.debug(
-      'OTEL_NODE_ENABLED_INSTRUMENTATIONS and OTEL_NODE_DISABLED_INSTRUMENTATIONS environment variables are mutually exclusive. All instrumentations are disabled.'
+      'OTEL_NODE_ENABLED_INSTRUMENTATIONS and OTEL_NODE_DISABLED_INSTRUMENTATIONS are both set.\nAny instrumentation that exists in both lists will be disabled.'
```

Comment:
nit:
```suggestion
      'OTEL_NODE_ENABLED_INSTRUMENTATIONS and OTEL_NODE_DISABLED_INSTRUMENTATIONS are both set. Any instrumentation that exists in both lists will be disabled.'
```

Multiline log message strings are a mild pain for processing (somewhat-)structured logs.  This is perhaps just personal preference, however.

Reply:
Per your other message... I'm totally onboard with dropping the debug altogether. I only added it when I was surprised by the disabling behavior. Now that I realize my brain was somewhere else that day, I agree that it doesn't help much. Will drop.

Reply:
Also good to know re: pain for logs. I didn't initially have this as two lines when I updated it, but wasn't sure if it was too long so I split it. But will keep in mind for future.